src/include/hip/rpp_hip_common.hpp

```diff
@@ -81,6 +87,12 @@ enum class RPPTensorDataType
 
 // schar
 typedef signed char schar;
+typedef struct d_schar3
```

Comment:
signed char is same as char. Can't we just use char3 built in type instead?

Reply:
That's true, but looks like it might be compiler dependent too - in the way they interpret it. 
So Rpp32s was chosen to be signed char at the top level rpp_defs, and that ends up following down to the low level API.

std::is_same<int, int>::value = 1
std::is_same<int, signed int>::value = 1 // implies compiler interpreted int is same as signed int
std::is_same<int, unsigned int>::value = 0 // implies compiler interpreted int is different from unsigned int

std::is_same<char, char>::value = 1
std::is_same<char, signed char>::value = 0 // implies compiler interpreted char is different from signed char
std::is_same<char, unsigned char>::value = 0 // implies compiler interpreted char is different from unsigned char